docs/keyboard.md

```diff
@@ -224,3 +224,51 @@ static scheduleLayoutAnimation(event)
 ```
 
 Useful for syncing TextInput (or other keyboard accessory view) size of position changes with keyboard movements.
+
+## Type Definitions
+
+### KeyboardMetrics
+`KeyboardMetrics` object is part of [`Keyboard`](text#addListener) callback and contains the measurement data of `Keyboard`.
+
+
+#### Example
+
+```js
+{
+  screenX: 0,
+  screenY: 401.1428527832031,
+  width: 411.4285583496094,
+  height: 282.28570556640625,
+}
+```
+
+#### Properties
+
+| Name      | Type    | Optional  | Description                                                         |
+| --------- | ------- | --------- | ------------------------------------------------------------------- |
+| screenX   | number  | No        | Virtual keyboard X position on screen in pixels.                    |
```

Comment:
Might reduce ambiguity to specify that we are referring to device-independent pixels instead of screen pixels.

```suggestion
| screenX   | number  | No        | Virtual keyboard X position on screen in CSS pixels.                    |
```